config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_logginglinks.logging.cnrm.cloud.google.com.yaml

```diff
@@ -39,7 +39,7 @@ spec:
       jsonPath: .status.conditions[?(@.type=='Ready')].lastTransitionTime
       name: Status Age
       type: date
-    name: v1alpha1
+    name: v1beta1
```

Comment:
Looks like we need the addition-versions label, so we continue to generate the alpha (assuming we shipped it)